susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -74,11 +74,14 @@ no_ssh_push_key_authorized:
     {%- set os_base = 'oracle' %}
   {%- endif %}
   #end of expections
+{%- elif grains['os_family'] == 'openEuler' %}
```

Comment:
To be honest, I didn't noticed yesterday, but this shoulkd be a separate if, so:
```suggestion
{%- endif %}

# openEuler Family. This OS is based in RedHat, but declares a separate family
{%- if grains['os_family'] == 'openEuler' %}
```